bin/main.rb

```diff
@@ -0,0 +1,85 @@
+require '../lib/board.rb'
+$winner = nil
+$current_player = "Player x"
+$win_combination = [
+  #vertical
+ [0,1,2],[3,4,5],[6,7,8],
+ #horizontal
+ [0,3,6],[1,4,7],[2,5,8],
+ #diagonal
+ [0,4,8],[2,4,6]
+]
+

```

Comment:
No **Global Variables** !!!

I suggest you to have 2 class, 
**Board** : where you could put `@win_combination`
**Game** : Where you could have `@winner` and `@current_player`

Reply:
done